### PR TITLE
feat: DIA-1144: SDK demo for mensuration and polling

### PR DESCRIPTION
**Pull request by matt-bernstein**

### CODE REVIEW

File: examples/mensuration_and_polling/poll_for_tasks.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    area_m = poly.area
+
+    perimeter_km = perimeter_m / 1e3
+    area_km = area_m / 1e6
+
+    return perimeter_km, area_km
+
+
+if __name__ == "__main__":
+    url = "http://localhost:8080"
```
Comment (niklub) [inline examples/mensuration_and_polling/poll_for_tasks.py:82]: is this script supposed to be used in the custom project setup? better to make reproducible setup and instructions for custom input parameters

File: examples/mensuration_and_polling/poll_for_tasks.py [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+            # currently, we only have one source image, but in general need to build a mapping between the task image (PNG) and the source image (TIFF)
+            perimeter, area = perimeter_and_area(
+                "data/e9b9661bcbd97b67f45364aafd82f9d6/response.tiff", annot
+            )
+            ls.tasks.update(
+                id=task.id,
+                data={**task.data, "label_perimeter": perimeter, "label_area": area},
+            )
+            print("updated task", task.id)
+    else:
```
Comment (niklub) [inline examples/mensuration_and_polling/poll_for_tasks.py:103]: do we need to keep it?

File: examples/mensuration_and_polling/README.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+1. Sign up at sentinel-hub.com for a trial account for access to [Sentinel-2](https://en.wikipedia.org/wiki/Sentinel-2) satellite images on demand. These have a resolution of 10 GSD (10 meters per pixel).
+2. Use the included `grab_georeferenced_image.py` with your credentials to download the image in a [UTM coordinate system](https://en.wikipedia.org/wiki/Universal_Transverse_Mercator_coordinate_system), where each pixel is a fixed area instead of a fixed fraction of longitude and latitude. This makes the image "square" with respect to the ground.
+3. Export to PNG:
+```bash
+convert data/e9b9661bcbd97b67f45364aafd82f9d6/response.tiff data/e9b9661bcbd97b67f45364aafd82f9d6/response.png
+```
+
+## Load data into Label Studio
+
+Create a new project and upload `data/e9b9661bcbd97b67f45364aafd82f9d6/response.png` to the project.
```
Comment (niklub) [inline examples/mensuration_and_polling/README.md:30]: can it be more readable path?

File: examples/mensuration_and_polling/grab_georeferenced_image.py [hunk 4]
```diff
@@ -0,0 +1,87 @@
+from sentinelhub import (
```
Comment (niklub) [inline examples/mensuration_and_polling/grab_georeferenced_image.py:1]: please add docstrings to each function explaining what it is suppose to be doing in this example.
also - are we sure all 3rd party dependencies are provided along the scripts? if not, please add installation instructions

**Review by niklub — CHANGES_REQUESTED** [review]

File: examples/mensuration_and_polling/poll_for_tasks.py (hunk 1 above)
Comment (matt-bernstein) [reply examples/mensuration_and_polling/poll_for_tasks.py:82]: No, this example is without custom scripting. Instructions for setup are in the readme

File: examples/mensuration_and_polling/poll_for_tasks.py (hunk 2 above)
Comment (matt-bernstein) [reply examples/mensuration_and_polling/poll_for_tasks.py:103]: I think it's a good example of how the same task is done with and without the new SDK, could help others migrating

File: examples/mensuration_and_polling/grab_georeferenced_image.py (hunk 4 above)
Comment (matt-bernstein) [reply examples/mensuration_and_polling/grab_georeferenced_image.py:1]: good point, will do this

File: examples/mensuration_and_polling/README.md (hunk 3 above)
Comment (matt-bernstein) [reply examples/mensuration_and_polling/README.md:30]: this is how the data is returned from sentinelhub along with the response.json, so it's also kind of a tutorial on how to do that. Can flatten the path if we don't care about how getting the imagery works.

File: examples/mensuration_and_polling/grab_georeferenced_image.py (hunk 4 above)
Comment (matt-bernstein) [reply examples/mensuration_and_polling/grab_georeferenced_image.py:1]: done

File: examples/mensuration_and_polling/README.md (hunk 3 above)
Comment (matt-bernstein) [reply examples/mensuration_and_polling/README.md]: done

**Review by niklub — APPROVED** [review]

**Pull request merged**